pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -990,3 +960,66 @@ func TestHighProirotyFlushUnschedulableQLeftover(t *testing.T) {
 		t.Errorf("Expected: %v after Pop, but got: %v", medPriorityPod.Name, p.Name)
 	}
 }
+
+// TestPodInfoInActiveQ tests that the pods with the same priority is sorted by the podInfo.timestamp.
+func TestPodInfoInActiveQComp(t *testing.T) {
```

Comment:
We need to test many more scenarios in this test, such as updating a pod in the activeQ, adding unschedulabe pod to the unschelableQ and then triggering a move request to the activeQ and ensuring that the timestamp is not updated, another similar test to check that when a pod goes to backoffQ and then the activeQ its timestamp remains the same, ....
You may want to take a look at [this test](https://github.com/kubernetes/kubernetes/pull/74041/files#diff-55c5c54bc3d25d4a1d157a644ffcd4aaR1130) to get ideas about how you can add many test scenarios involving various operations in this test.

Reply:
Thanks for your help. I add many more scenarios in this test about activeQ/unschedulabeQ/backoffQ. If there's anything else missing, you can comment and I'll add it as soon as possible.